Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -293,18 +290,15 @@ void EmbeddedFrag::updateNewNeighs(
 
   d_eatoms[aid].neighs.clear();
   RDKit::INT_VECT hIndices;
-  RDKit::ROMol::ADJ_ITER nbrIdx, endNbrs;
-  boost::tie(nbrIdx, endNbrs) =
-      dp_mol->getAtomNeighbors(dp_mol->getAtomWithIdx(aid));
-  while (nbrIdx != endNbrs) {
-    if (d_eatoms.find(*nbrIdx) == d_eatoms.end()) {
-      if ((*dp_mol)[*nbrIdx]->getAtomicNum() != 1) {
-        d_eatoms[aid].neighs.push_back(*nbrIdx);
+  for (auto nbrIdx : boost::make_iterator_range(
+           dp_mol->getAtomNeighbors(dp_mol->getAtomWithIdx(aid)))) {
```

Comment:
Here too

Reply:
same story: always need the index, sometimes need the atom